### PR TITLE
Crear componente para renderizar sección "Nosotros" con información de la ONG

### PR DESCRIPTION
**Pull request by ArielSurco**
User story link: https://alkemy-labs.atlassian.net/browse/OT91-54

**##SUMMARY**
It show the top section of "About us" component in /nosotros route

**##CHANGES ADDED**
- Render AboutUs in /nosotros route
- Add AboutUsDescription component: Use Title component, displays "Nosotros" and receives a "description" property that is displayed below the title.
- Add AboutUs styles
- Use description const in AboutUs.js: Represents a description obtained with an API request and uses it to render the AboutUsDescription component

**##SCREENSHOTS**
image

### CODE REVIEW

File: src/Components/Title/Title.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useState, useEffect } from "react";
 import Box from "@material-ui/core/Box";
 
-import DefaultImage from "../../Assets/Title/default.jpg";
+import DefaultImage from "../../assets/Title/default.jpg";
```
Comment (saralavanchy) [inline src/Components/Title/Title.js]: this file is not scope of this PR

File: src/Components/Newsletter/NewsletterForm.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const SetTokenNewsLetter = () =>{
     localStorage.setItem("tokenNL","settokenNL")
 }
```
Comment (saralavanchy) [inline src/Components/Newsletter/NewsletterForm.js:6]: this file is not scope of this PR

File: src/Components/Users/UserModifierForm.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const UserModifierForm = ({user}) => {
             return true;
         };
```
Comment (saralavanchy) [inline src/Components/Users/UserModifierForm.js:30]: this file is not scope of this PR

File: src/Components/utilsData/roleList.js [hunk 4]
```diff
@@ -1 +1 @@
-export const rolesList = [{roleId:'Administrador'},{roleId:'Regular'}]
\ No newline at end of file
+export const rolesList = [{roleId:'Administrador'},{roleId:'Regular'}]
```
Comment (saralavanchy) [inline src/Components/utilsData/roleList.js:1]: this file is not scope of this PR

File: src/Services/userServices.js [hunk 5]
```diff
@@ -8,4 +8,4 @@ export const createUser = async (user) => {
 export const updateUser = async (id,user) => {
     const {data} = await axios.put(`http://ongapi.alkemy.org/api/users/${id}`,user)
     return data
-}
```
Comment (saralavanchy) [inline src/Services/userServices.js:11]: this file is not scope of this PR

File: src/Styles/AboutUs.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
+.about-us-description {
```
Comment (saralavanchy) [inline src/Styles/AboutUs.css]: try to improve this file thinking on make this styles reusable

Comment (saralavanchy) [reply src/Styles/AboutUs.css]: if this is a container, create a container file for styles and put this in it. Try to never use a particular stylesheet for a component.

File: src/Components/Activities/ActivitiesForm.js [hunk 7]
```diff
@@ -15,7 +15,7 @@ const ActivitiesForm = () => {
       setInitialValues({ ...initialValues, description: e.target.value });
```
Comment (saralavanchy) [inline src/Components/Activities/ActivitiesForm.js]: this file is not scope of this PR

File: src/Components/About/AboutUsDescription.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import { Box, Typography } from "@material-ui/core";
+import Title from "../Title/Title";
+import '../../Styles/AboutUs.css';
+
+const AboutUsDescription = ({description}) => {
+    return (
+        <Box className="about-us-description">
+            <Title title="Nosotros"/>
```
Comment (saralavanchy) [inline src/Components/About/AboutUsDescription.js]: the title should be on tje About Us Component and not on the description component

File: src/Components/About/AboutUsDescription.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+            <div className="about-us-description__text">
```
Comment (saralavanchy) [inline src/Components/About/AboutUsDescription.js]: use the same logic for the names: about-us-description-text

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/Title/Title.js (hunk 1 above)
Comment (ArielSurco) [reply src/Components/Title/Title.js]: I made that change to fix the compile problem.
The other changes out of scope were eslint fixes, I have to solve it with a merge from development to this branch, right?

**Pull request closed without merging**